tests/reducers/settingEncounterTypeReducer.test.js

```diff
@@ -36,17 +36,7 @@ describe('Setting Encounter Type reducer', () => {
         );
         expect(actualState).toEqual(expectedState);
     });
-    it('should handle `SETTING_ENCOUNTER_TYPE_LOADING`', () => {
```

Comment:
Why delete this too?

Reply:
Similarly, the `it` block calls `settingEncounterTypeLoading` in `settingEncounterType.js` action which is no longer used in the app.